src/Evolution/Systems/Cce/Tags.hpp

```diff
@@ -124,7 +124,7 @@ struct Dy : db::PrefixTag, db::SimpleTag {
 /// The derivative with respect to Bondi \f$r\f$
 template <typename Tag>
 struct Dr : db::PrefixTag, db::SimpleTag {
-  using type = Scalar<SpinWeighted<ComplexDataVector, Tag::type::type::spin>>;
+  using type = db::item_type<Tag>;
```

Comment:
why this change?

Reply:
In the new `WorldtubeData` class, I represent the (Bondi-Sachs) radial derivative of tensor quantities like the spatial metric, so it doesn't work to assume that it prefixes a spin-weighted quantity anymore.

Reply:
just make it `typename Tag::type`